docs/source/host-collect-analyze/cpu.md

```diff
@@ -0,0 +1,67 @@
+---
+title: CPU
+description: Collect and analyze information about the number of CPU cores
+---
+
+## CPU Collector
+
+The `cpu` collector can be used to collect information about the number of CPU cores on a host
+
+### Parameters
+
+`None`
```

Comment:
Same question as the other one - if they don't literally enter `None`, suggest changing the formatting.